src/samplers/cpu/linux/usage/bpf.rs

```diff
@@ -142,34 +163,34 @@ impl CpuUsage {
         // refresh the counters from the kernel-space counters
         self.bpf.refresh_counters(elapsed);
 
-        // get the new sum of all the counters
-        let sum_now: u64 = sum();
-
-        // get the number of nanoseconds in busy time, since idle hasn't been
-        // incremented, the busy time is the difference between our prev and
-        // current sums
-        let busy_delta = sum_now.wrapping_sub(self.sum_prev);
+        // update busy time metric
+        let busy: u64 = busy();
+        let busy_prev = CPU_USAGE_BUSY.value();
+        let busy_delta = busy.wrapping_sub(busy_prev);
+        self.total_busy.set(elapsed.as_secs_f64(), busy);
```

Comment:
I get why you're reading from `CPU_USAGE_BUSY` and then writing to `self.total_busy` so that you don't have to handle the histogram separately, but it is a bit confusing to read. Is it worth adding a `get` or `value` to the `Counter` so that we could just call `self.total_busy.value` instead?

Reply:
This is fixed too